R/zzz.R

```diff
@@ -79,6 +79,13 @@
     vctrs::s3_register("dplyr::dplyr_reconstruct", "resample_results", method = dplyr_reconstruct_resample_results)
     vctrs::s3_register("dplyr::dplyr_reconstruct", "iteration_results", method = dplyr_reconstruct_iteration_results)
   }
+
+  if (newer_tibble()) {
+    vctrs::s3_register("tibble::obj_sum", "tune_results")
+    vctrs::s3_register("tibble::obj_sum", "resample_results")
```

Comment:
You don't need to conditionally register these two. They will automatically be registered since you `#' @export` them and did `@importFrom tibble obj_sum`